src/app/item-triage/ItemTriage.tsx

```diff
@@ -111,19 +110,19 @@ export function ItemTriage({ item }: { item: DimItem }) {
         title="In Loadouts"
         sectionId="triage-loadout"
         defaultCollapsed={true}
-        extra={inLoadouts.length}
+        extra={inLoadouts?.length}
         extraOnlyCollapsed
-        disabled={!inLoadouts.length}
+        disabled={!inLoadouts?.length}
       >
         <ul>
-          {inLoadouts.map((l) => (
-            <li className={styles.loadoutRow} key={l.id}>
-              {l.name}{' '}
+          {inLoadouts?.map((l) => (
+            <li className={styles.loadoutRow} key={l.loadout.id}>
```

Comment:
Not for this PR but I feel like these could be larger with some padding
<img width="388" alt="Screen Shot 2022-04-27 at 8 46 33 pm" src="https://user-images.githubusercontent.com/7344652/165501894-f310c93c-a054-4b2e-b077-c9dc8caac1fc.png">

Reply:
this branch was split way before latest triage stuff